base/tks/src/org/dogtagpki/server/tks/servlet/TokenServlet.java

```diff
@@ -2942,15 +2939,14 @@ private void processComputeSessionKeysSCP03(HttpServletRequest req, HttpServletR
                     keyNickName, // KeyNickName
                     keySet, // TKSKeyset
                     log_string_from_keyInfo(xkeyInfo), // KeyInfo_KeyVersion
+                    null,
+                    null,
```

Comment:
Same problem here.

Reply:
This is for both of these instances. Those 2 values were left out due to not quite finishing the code that uses those values for scp03. With the incoming patches this will be resolved, so leave placeholders for them here and they will be used later.